test/1.4/lib/T_signal_templates.py

```diff
@@ -53,12 +53,14 @@ def signal_lower(self):
 obj.outsig = stub
 stest.expect_equal(stub.level, 1)
 
-# reverse execution doesn't cause signal pulsing
-transitions = stub.transitions
-simics.SIM_run_command('enable-reverse-execution')
-simics.SIM_run_command('continue 1000')
-simics.SIM_run_command('reverse')
-stest.expect_equal(stub.transitions, transitions)
+# Reverse execution is removed in Simics 7
+if int(simics.SIM_version_major()) < 7:
```

Comment:
This'll crash if the version in play is 4.8, won't it? Not that I expect that ever happens, but still, for the sake of principle I'd suggest:
`if simics.SIM_version_major() == '4.8' or int(simics.SIM_version_major()) < 7:`

Reply:
Would  `int(simics.SIM_version_major().split(".")[0]) < 7` be okay? Less clean but works with outdated Simics versions.

Reply:
When I think more about it I think DML 1.4 was added in 6, maybe with some tech preview in 5. I don't think it's worth adding the extra ugliness for supporting this test to be allowed to run in old Simics versions with "x.y" major versions.

Reply:
DMLC in principle still supports 4.8. But yes, this is nit-picky, and I only brought it out of principle's sake. It's not a blocker.

Reply:
Actually... rather, what we support is _API_ version 4.8. I'm not sure what majors of Simics proper we support. This makes this more shaky.

Reply:
I changed the check to see if rev-execution class is available or not.